Automattic/simplenote-ios
# Drops SAMKeychain: Mark 2

### Fix
- Replaces **SAMKeychain** usage with **KeychainManager**
- Drops the **SAMKeychain** dependency!! 🥁 

@aerych Sir!! May I bug you with the last one in these series?
YAY one less dependency!!!

Closes #880


---

### Test: Upgrade Path
1. Install `develop` and log into your account
2. Build this branch
3. Launch the `SimplenoteShare` Scheme against Safari
4. Trigger the Share Sheet

- [x] Verify the Simplenote Extension shows up
- [x] Verify that the Share Extension **can** post a new note into your account

### Test: Fresh Install
1. Fresh install this branch
2. Log into your account
3. Open Safari and trigger the Share Sheet

- [x] Verify the Simplenote Extension shows up
- [x] Verify that the Share Extension **can** post a new note into your account

### Release
These changes do not require release notes.
